md/BonitaBPM_platform_setup.md

```diff
@@ -102,27 +109,69 @@ If you want to modify the configuration of an already initialized Bonita BPM Pla
 Note that the *Platform Setup tool* does not need the Bonita BPM Server to be running for the configuration to be updated. However, the server needs to be restarted for the changes to take effect.
 :::
 
-
 ::: warning
 Keep in mind that the folder **`platform_conf/initial`** is not used anymore once the platform has been initialized for the first time. To update your configuration, only the **`platform_conf/current`** folder is taken into account.
 :::
 
-## Troubleshooting
 
----
+<a id="run_bundle_configure" />
+
+## Run the Bonita BPM bundle configuration (Tomcat 7 / WildFly 9)
+
+::: info
+In the [Tomcat](tomcat-bundle.md) and [WildFly](wildfly-bundle.md) bundles, configuration is done automatically when calling `bonita-start.sh / .bat`
+:::
+
+1. Update the file `database.properties` with the database properties matching your environment (db vendor name, server name, server port, connection username and password)
+2. **Oracle and SQL Server only**: add your JDBC drivers in `setup/lib` folder (PostgreSQL, MySQL, and H2 drivers are already shipped with the tool)
+3. From the tool folder, run `./setup.sh configure` (Unix/Mac) or `setup.bat configure` (Windows).
 
-**Problem**: When I run Platform setup tool, I get the exception: `locked by an other process`
 
-**Possible cause**: The Platform setup tool tries to access the same database (H2) as a running Bonita BPM Bundle or Bonita BPM Studio.
+::: info
```

Comment:
This page is already for advanced users, so someone reading it knows what they are doing